test/acceptance/tests/basic/basic_test.go

```diff
@@ -22,127 +23,150 @@ func TestValidation_RetryJoinRequired(t *testing.T) {
 		TerraformDir: "./terraform/retry-join-validate",
 		NoColor:      true,
 	})
-	defer terraform.Destroy(t, terraformOptions)
+	defer terraform.DestroyE(t, terraformOptions)
```

Comment:
Needed to add this because it was failing for me locally because `terraform.Destroy` calls `require.NoError` and `terraform destroy` would have the same error as the apply. I'm not sure why it works in CI (perhaps a different version of TF?)